reporter/src/main/kotlin/Reporter.kt

```diff
@@ -53,15 +54,15 @@ abstract class Reporter {
 
     /**
      * Generate a report for the [ortResult], taking into account any issue resolutions provided by [resolutionProvider]
-     * and the given known [copyrightGarbage]. The report may be post-processed by a [postProcessingScript] before it is
-     * written to [outputStream].
+     * and any license text provided by [LicenseTextProvider]. Copyright statements are cleaned from [copyrightGarbage].
+     * The report may be post-processed by a [postProcessingScript] before it is written to [outputStream].
      */
     abstract fun generateReport(
-        ortResult: OrtResult,
-        resolutionProvider: ResolutionProvider,
-        licenseTextProvider: LicenseTextProvider,
-        copyrightGarbage: CopyrightGarbage,
         outputStream: OutputStream,
+        ortResult: OrtResult,
```

Comment:
I would have preferred to have `ortResult` above `outputStream` to have the input before the output, but approved anyway.

Reply:
Thanks. I was considering that, but in the end I preferred to have `ortResult` next to the arguments that modify / influence it.